fastapi/responses.py

```diff
@@ -17,7 +17,19 @@
 
 class ORJSONResponse(JSONResponse):
     media_type = "application/json"
+    option: int = 0
```

Comment:
why not just extend the `__init__` ?

```python
    def __init__(self, option, **kwds):
        self.option = option
        super().__init__(**kwds)
```

Reply:
@raphaelauv I need somehow to store option so that when ORJson response is being initiated, it must get option from context not from constructor